flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
and for the record it is still disappointing that RaspberryPi users are left out of this. Programming Arduino and using a RespberryPi goes hand in hand.

Reply:
agreed. but there isn't much i can do about this except for wait for support for arm based cpu's.

Reply:
building from source instead of pre-built binaries.

Reply:
but I repeated that comment since you closed the previous PR.

Reply:
> building from source instead of pre-built binaries.

yeah i will consider this once i can get the application to compile for x86_64 first, but with support for arm still being experimental at best i dont believe its best to add it until the devs deem it almost ready. This is one of the reasons making a new app on flathub is a good idea as users may still want to use the old ide especially on arm.